tmpfiles.sh

```diff
@@ -253,7 +401,12 @@ _D() {
 	local path=$1 mode=$2 uid=$3 gid=$4
 
 	if [ -d "${path}" ] && [ "${REMOVE}" -gt 0 ]; then
-		dryrun_or_real find "${path}" -mindepth 1 -maxdepth 1 -xdev -exec rm -rf {} +
+		if owned_by_root "${path}" ; then
+			echo "Cowardly refusing to remove directory" >&2
```

Comment:
doesn't this defeat the purpose of using tmpfiles.d ?  the entries are supposed to get arbitrary filesystem state back into a known good state.  by bailing out here (and elsewhere), the project is only good to create files when they don't yet exist.

Reply:
The coward refusal happens only when the owner is root.
Removing root-owned directories implies a potential security risk.
user-owned directories are allowed to be removed.